### DISCUSSION TITLE
Asset validation failed (90022) and (90704)

### DISCUSSION THREAD

**Discussion by ZhorovEgor** [airsdk/Adobe-Runtime-Support#3158] [General]
Hellow! After piblish project on Adobe Animate AIR by Harmon and deploy project to the TRANSPORTER i have this error:

Asset validation failed (90022)
Missing required icon file. The bundle does not contain an app icon for iPhone / iPod Touch of exactly '120x120' pixels, in .png format for iOS versions >= 10.0. To support older versions of iOS, the icon may be required in the bundle outside of an asset catalog. Make sure the Info.plist file includes appropriate entries referencing the file. See https://developer.apple.com/documentation/bundleresources/information_property_list/user_interface (ID: e9a3cb0c-050f-439d-b254-2520f1add472)

Asset validation failed (90704)
Missing app icon. Include a large app icon as a 1024 by 1024 pixel PNG in the asset catalog of apps built for iOS, iPadOS, or watchOS. Without this icon, apps can't be submitted for review. For details, visit: https://developer.apple.com/documentation/xcode/managing-assets-with-asset-catalogs. (ID: 195275c1-3a9e-4167-b2f5-4157c500c399)

But i have png icon on project ant all other. Something is wrong?

![photo_2024-04-04_20-05-50](https://github.com/airsdk/Adobe-Runtime-Support/assets/63095086/bdbcfa71-116d-4ecf-8055-a1fa225b1fac)

![photo_2024-04-04_20-06-45](https://github.com/airsdk/Adobe-Runtime-Support/assets/63095086/75cd0611-3c01-4055-bcf9-978b1ac6deb1)

**Comment by jeuxdaily** [comment]
hello
I have the same problems
Are you solve it

ا

**Reply by ZhorovEgor** [reply]
Howdy! Yap i solved. Just create assets.car on some site and input file on project

**Reply by jeuxdaily** [reply]
That's exactly what I did.
thanks
good luck

**Reply by jeuxdaily** [reply]
is there any method to download IPA file to pc

**Reply by pol2095** [reply]
I never tried but you can download [Transporter for Windows](https://help.apple.com/itc/transporteruserguide/en.lproj/static.html#apdA74e239d7) (it's a command line tool).

**Reply by jeuxdaily** [reply]
Thanks I will try the site
